recipe/meta.yaml

```diff
@@ -27,18 +27,19 @@ requirements:
     - wheel
   run:
     - anytree >=2.8.0
-    - autograd >=1.6.2
-    - casadi >=3.6.3
+    - casadi >=3.6.5
     - imageio >=2.3.0
     - matplotlib-base >=3.6.0
-    - numpy >=1.23.5
+    - numpy <=2
```

Comment:
less than or equal to 2? should this be strictly less than?

Reply:
Oops, thank you.

```suggestion
    - numpy <2
```